src/norm_layer_cuda.cu

```diff
@@ -1089,8 +1089,10 @@ void LayerNormCuda::forward(BaseHiddenStates &input_states,
     TempStateCuda *cu_temp_states = dynamic_cast<TempStateCuda *>(&temp_states);
```

Comment:
could you please remove this `cu_temp_states` as well? because it is unused

Reply:
Done.